bundles/org.openhab.binding.ondilo/README.md

```diff
@@ -0,0 +1,122 @@
+# Ondilo Binding
+
+This binding integrates Ondilo ICO pool monitoring devices with openHAB, allowing you to monitor and automate your pool environment using openHAB’s rules and UI.
+
+## Supported Things
+
+`bridge:` Represents your Ondilo account (OAuth2)
+

```

Comment:
```suggestion

```

Reply:
removing the line with the bridge was the intention here?